<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurvatureAnisotropicDiffusionFilterInterface</class>
 <widget class="QWidget" name="CurvatureAnisotropicDiffusionFilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>213</width>
    <height>306</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anisotropic Diffusion Filter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CurvatureAnisotropicDiffusionFilterInterface{
	border-radius: 3px;
	border: 1px solid rgb(80, 80,80);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="target_groupbox_">
       <property name="title">
        <string>Target Layer</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QComboBox" name="target_layer_"/>
        </item>
        <item>
         <widget class="QCheckBox" name="use_active_layer_">
          <property name="text">
           <string>Always Use Active Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="preserve_data_format_">
          <property name="text">
           <string>Preserve Data Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iterationsHLayout_top">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Number of Iterations</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="iterationsHLayout_bottom"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sensitivityHLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Sensitivity Range</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="sensitivityHLayout_bottom"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="replaceRunWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="replaceCheckBox">
        <property name="text">
         <string>Replace</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runFilterButton">
        <property name="text">
         <string>Run Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
